Workbooks/AKS/Syslog/Syslog.workbook

```diff
@@ -291,7 +293,7 @@
                 {
                   "operator": "Default",
                   "thresholdValue": null,
-                  "representation": "success",
+                  "representation": "Normal",

```

Comment:
For the lines above: The thresholdValue needs to be renamed based on what we have here:

project Emergency=emerg, Alert=alert, Critical=crit, Error=err, Warning=err, Warning=warning, Notice=notice, Informational=info, Debug=debug",


       {
                  "operator": "==",
                  "thresholdValue": "info",
                  "representation": "1",
                  "text": "{0}{1}"
                },
                {
                  "operator": "==",
                  "thresholdValue": **"warn",** (This needs to be renamed to warning)
                  "representation": "2",
                  "text": "{0}{1}"
                },
                {
                  "operator": "==",
                  "thresholdValue": "err",
                  "representation": "3",
                  "text": "{0}{1}"
                },

Reply:
Updated

Reply:
For this update, I meant for Line 239 (The tile part -Summary by Severity Level). Looks like you updated the one under 'Raw syslog records'. Can you revert the changes you made for the 'Raw syslog records' and update the one in the tiles part?

Reply:
Alright, I just realized for the query part, this part below is commented out. Can you remove the line if it is not required? (At line 216)

`project Emergency=emerg, Alert=alert, Critical=crit, Error=err, Warning=err, Warning=warning, Notice=notice, Informational=info, Debug=debug"`

Can you also make sure for line 239-292 and line 492-545, the threshold value equals to the correct SeverityLevel?

- info
- notice
- warning
- error
- (I am not sure about the other values like critical, notice, emergency, etc)

And at line 543, make sure the default representation is not 'success'